proto/cosmos/evidence/v1beta1/evidence.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package cosmos.evidence.v1beta1;
 
-option go_package            = "github.com/cosmos/cosmos-sdk/x/evidence/types";
+option go_package            = "cosmossdk.io/x/evidence/types";
```

Comment:
We don't need to change the go_package value to the vanity URL, the old one should still be valid too.   I think these errors are valid, WDYT @amaurym ?

Reply:
It is quite confusing if we don't change it, imho.